packages/@snowpack/plugin-workbox/README.md

```diff
@@ -0,0 +1,21 @@
+# @snowpack/plugin-workbox
+
+<!-- Use Webpack to bundle your application for production. -->
```

Comment:
WDYT about adding a description that adds a link to [Workbox](https://developers.google.com/web/tools/workbox)? That way people can read a little more about it (I say that not being familiar with it myself).